<launch>
    <arg name="pkg" value="quickdev_examples" />
    <arg name="type" value="message_array_cache" />
    <arg name="name" default="$(arg type)" />
    <arg name="rate" default="10" />
    <arg name="args" value="_loop_rate:=$(arg rate) ~string:=/multi_publisher/string" />
    <node
        pkg="$(arg pkg)"
        type="$(arg type)"
        name="$(arg name)"
        args="$(arg args)"
        output="screen" />
    <include file="$(find quickdev_examples)/launch/multi_publisher.launch">
        <arg name="rate" value="$(arg rate)" />
    </include>
</launch>
